src/cljam/util/chromosome.clj

```diff
@@ -13,6 +13,11 @@
   [s]
   (cstr/replace s #"(?i)^chr" ""))
 
+(defn- split-version-suffix
+  [s]
+  (let [[_ base suffix leftover] (re-matches #"(.+?)((?i)v[0-9](_alt|_random)?)?$" s)]
```

Comment:
`leftover`  is never used, please replace it with `_`. `$` is not needed for `re-matches`.